html/about.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link rel="stylesheet" href="/css/about.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/716817a6e7.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="page-heading">
+        <i class="fa fa-facebook-f"></i>
+        <i class="fa fa-twitter"></i>
+        <span class="heading-label">English</span>
+        <span class="heading-label">My Page</span>
+        <span class="heading-label">Logout</span>
+      </div>
+      <div class="navbar">
+        <a href="../html/index.html"
+          ><img
+            src="/images/logo.png"
+            alt="Sawasawa brand logo"
+            class="logo-img"
+        /></a>
+        <nav>
+          <div class="nav-links" id="navLinks">
+            <i class="fas fa-times" id="closeBtn"></i>
+            <ul class="nav-items" id="navItems">
+              <li><a href="../html/index.html">About</a></li>
```

Comment:
- [ ] Good job adding the link to the index.html page so users can easily navigate back to the home page please change text shown from About to Home